README.md

```diff
@@ -77,3 +78,6 @@ Install it with:
 `sudo gem install gherkin_lint`
 
 After that `gherkin_lint` executable is available.
+
+## Configuration
+If you have a custom configuration you'd like to run on a regular basis instead of passing enable and disable flags through the CLI on every run, you can configure a ```.gherkin_lint.yml``` file that will be loaded on execution.  The format and available linters are in [```config/default.yml```](config/default.yml)
```

Comment:
I wrote a feature file to demonstrate that this is working as you specified.  Is this okay?  We are already doing what you asked, I think.  I just wrote a test to prove it.